internal/models/user_submission.go

```diff
@@ -0,0 +1,17 @@
+package models
+
+import "time"
+
+type UserSubmission struct {
+    ID            uint      `gorm:"primaryKey" json:"id"`
```

Comment:
we are using UUID v7 for all primary keys

Reply:
done